server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -268,12 +269,13 @@ class DeltaSharingService(serverConfig: ServerConfig) {
 
     val start = System.currentTimeMillis
     val tableConfig = sharedTableManager.getTable(share, schema, table)
-    if (queryTableRequest.version.isDefined) {
+    if (queryTableRequest.version.isDefined || queryTableRequest.timestamp.isDefined) {
       if (!tableConfig.cdfEnabled) {
-        throw new DeltaSharingIllegalArgumentException("Reading table by version is not supported" +
-          s" because change data feed is not enabled on table: $share.$schema.$table")
+        throw new DeltaSharingIllegalArgumentException("Reading table by version or timestamp is" +
+          " not supported because change data feed is not enabled on table: " +
```

Comment:
Question: With streaming, we will allow time travel queries even if cdf is not enabled... right?

Reply:
Yes. We'll have another field to guard this, pending on the decision about default behavior change.